enterprise/frontend/src/metabase-enterprise/license/components/BillingInfo/BillingInfo.styled.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { color } from "metabase/lib/colors";
+import Card from "metabase/components/Card";
+import { Text, Icon } from "metabase/ui";
+import Link from "metabase/core/components/Link";
+
+export const BillingErrorContainer = styled.div`
+  margin-top: 0.5rem;
+  white-space: nowrap;
+  color: ${color("error")};
+`;
+
+export const BillingErrorMessage = styled.div`
+  margin-top: 1rem;
+`;
```

Comment:
nit: I think both of these could be mantine Box components with some [style props](https://v6.mantine.dev/styles/style-props/)